src/main/scala/mesosphere/marathon/core/flow/impl/ReviveOffersActor.scala

```diff
@@ -37,6 +38,8 @@ private[impl] class ReviveOffersActor(
     offersWanted: Observable[Boolean],
     driverHolder: MarathonSchedulerDriverHolder) extends Actor with StrictLogging {
 
+  private[this] val revivedOffersMetric = Metrics.minMaxCounter(ServiceMetric, getClass, "revivedOffers")
```

Comment:
Why do we use the class name for metrics name? Does it fit in our current schema? I mean, for a marathon user is `mesosphere.marathon.core.flow.impl.ReviveOffersActor.revivedOffers` pretty much non-sensical as metrics-prefix.

Reply:
Yeah, I know. We are already doing this already everywhere, and changing this would be a breaking change to our consumers.

Reply:
So we use the class prefix :sigh: but `revivedOffers` sounds like this is the number of offers revived. However, this is the number of times we send `Revive` command to Mesos. So maybe smth. like `numOfferRevivals`?

Reply:
I agree... how about `reviveCount`

Reply:
`reviveCount` works for me 👍